src/tendril.rs

```diff
@@ -906,7 +1062,11 @@ impl<F> Tendril<F>
     }
 }
 
-impl DerefMut for Tendril<fmt::Bytes> {
+// There's no need to worry about locking on an atomic Tendril, because it makes it unique as
+// soon as you do that.
```

Comment:
I will admit that I didn’t check what this was doing; I had made assumptions about what it *would* do, assumptions that by the sound of it may be wrong. I guess we’d better check that.

A `make_owned` could then in turn be called by `into_send`.

For comparison, `Rc` has `make_unique`. I think `make_owned` is still a better name for us here as that’s the aspect of the semantics that we focus on.